doc/source/examples/extended_examples/sfem/stochastic_fem.rst

```diff
@@ -0,0 +1,434 @@
+variance should converge towards theoretical values calculated in :math:numref:`theoretical mean` and :math:numref:`theoretical variance`.
+
+Firstly, several realizations are generated. 5000 is enough, which is the same as the number of simulations to be run later on. Statistical processing
+is then performed on these realizations.
+
+.. literalinclude:: sfem.py
+  :language: python
+  :lines: 350-385
+
+You can generate a plot of the mean versus the number of realizations:
+
+.. literalinclude:: sfem.py
+  :language: python
+  :lines: 387-397
+
+.. figure:: mean.png
+   
+   The mean converges to the true value as the number of realizations is increased.
+
+You can also generate a plot of the variance versus the number of realizations:
+
+.. literalinclude:: sfem.py
+  :language: python
+  :lines: 399-409
+
+.. figure:: variance.png
+
+  The variance converges to the true value as the number of realizations is increased.
+
+The preceding plots confirm that the implementation is correct. If you desire more accuracy, you can further decrease the minimum eigenvalue, but the value already chosen is sufficient.
+
+Running the simulations
+~~~~~~~~~~~~~~~~~~~~~~~
+Focus now shifts to the PyMAPDL part of this example. Remember that the problem requires running 5000 simulations. Therefore,
+you must write a workflow that performs these actions:
+
+1. Create the geometry of the cantilever model.
+
+2. Mesh the model. The following code uses the four-node PLANE182 elements.
+
+3. For each simulation, generate one realization of :math:`E` and one sample of :math:`P`.
+
+4. For each simulation, loop through the elements and for each element, use the generated 
+   realization to assign the value of the Young's modulus. Also assign the load for each simulation.
+
+5. Solve the model and store :math:`u` for each simulation.
+
+.. note::
+  One realization continuously varies with :math:`x`, but a plane stress element like PLANE182 can only have a constant
+  Young's modulus assigned. Therefore, for an element whose :math:`x`-coordinates are between :math:`x_1` and :math:`x_2`, you can simply
+  assign the average value of :math:`E` between these two values or assign the value of :math:`E` at the centroid. The later is
+  chosen for this implementation. The method chosen becomes insignificant with a finer mesh as both methods should produce similar
+  results.
+
+This function implements the preceding steps:
+
+.. literalinclude:: sfem.py
+  :language: python
+  :lines: 444-556
+
+You can pass the required arguments to the defined function to run the simulations:
+
+.. literalinclude:: sfem.py
+  :language: python
+  :lines: 559-569
+
+Answering problem questions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To determine the PDF of the response :math:`u`, you can perform a statistical post-processing of
+simulation results:
+
+.. literalinclude:: sfem.py
+  :language: python
+  :lines: 571-582
+
+.. figure:: pdf.png
+
+   The PDF of response :math:`u`
```

Comment:
Is this a caption? If so, it should have a concluding period. If not, words seem to be missing.